api_testing/apis/test_plan_api.py

```diff
@@ -286,6 +286,34 @@ def get_suborder_data_with_different_material_type(self, material_type):
 
         return test_plan, test_unit
 
+    def create_completed_testplan_multiple_testunits(self) :
```

Comment:
you can update` "create_completed_testplan_random_data"` that already on master by this 
https://github.com/modeso-open-source/1lims-automation/pull/376/files#diff-3ff37bea6803a6a7ceb457c30245ac2cR323
and use it in your branch instead of `create_completed_testplan_multiple_testunits`

Reply:
done